src/amuse/ext/orbital_elements.py

```diff
@@ -10,10 +28,9 @@
 
 from amuse.units.quantities import to_quantity, VectorQuantity
 
-
 def newton(f, x0, fprime=None, args=(), tol=1.48e-8, maxiter=50):
     if fprime is None:
-        print("provide fprime")
+        warnings.warn("provide fprime")
```

Comment:
Not sure either, seems better to have it as a warning indeed. Anyway, you wrote it ;).